docs/paypal.md

```diff
@@ -0,0 +1,173 @@
+# Pay with PayPal
+
+Accept PayPal payments in your iOS app using the PayPal Payments SDK.
```

Comment:
We haven't landed on a name for this SDK yet so have used a few different names throughout docs/project. We should settle on a differentiated name for the SDK as well as the PayPal module. Currently we call the SDK `PayPal` and also have a `PayPal` module for the PayPal checkout flow.